src/shogun/evaluation/SplittingStrategy.cpp

```diff
@@ -77,9 +77,8 @@ SGVector<index_t> CSplittingStrategy::generate_subset_indices(index_t subset_idx
 {
 	if (!m_is_filled)
 	{
-		SG_ERROR("Call %s::build_subsets() before accessing them! If this error"
-				" stays, its an implementation error of %s::build_subsets()\n",
-				get_name(), get_name());
+		SG_ERROR("Call build_subsets() before accessing them! If this error"
```

Comment:
We generally try to avoid method names in errors. This is not your fault here, but you could clean it up:

"Subsets need to be built before accessing them.\n"